src/man/firejail.txt

```diff
@@ -2030,6 +2030,14 @@ For details, see
 #903
 .UE
 
+.TP
+\fB\-\-keep-shell-rc
+.br
+By default, when using a private home directory, firejail creates files based
+on the operating system home "skeleton". (i.e. ~/.bashrc, ~/.bash_profile, ...).
+This option disables this feature, and enables the user to whitelist these
+files as original files.
```

Comment:
```suggestion
By default, when using a private home directory, firejail copies files from the
system's user home template (/etc/skel) into it, which overrides attempts to
whitelist the original files (such as ~/.bashrc and ~/.zshrc).
This option disables this feature, and enables the user to whitelist the
original files.
```

Edit to make it clearer.

Also, note that `skel` does not actually copy .bash_profile (which is intended
for login shells), only rc files.